modules/skybrowser/skybrowsermodule.cpp

```diff
@@ -0,0 +1,692 @@
+            },  
+            {
+                "centerTargetOnScreen",
+                &skybrowser::luascriptfunctions::centerTargetOnScreen,
+                "string",
+                "Takes an identifier to a sky browser and animates its corresponding "
+                "target to the center of the current view."
+            }, 
+            {
+                "setImageLayerOrder",
+                &skybrowser::luascriptfunctions::setImageLayerOrder,
+                "string, int, int",
+                "Takes an identifier to a sky browser or a sky target, an image index "
+                "and the order which it should have in the selected image list. The "
+                "image is then changed to have this order."
+            }, 
+            {
+                "addPairToSkyBrowserModule",
+                &skybrowser::luascriptfunctions::addPairToSkyBrowserModule,
+                "string, string",
+                "Takes the identifier of the sky target and a sky browser and adds them "
+                "to the sky browser module."
+            },
+            {
+                "setEquatorialAim",
+                &skybrowser::luascriptfunctions::setEquatorialAim,
+                "string, double, double",
+                "Takes the identifier of a sky browser or a sky target and equatorial "
+                "coordinates Right Ascension and Declination. The target will animate to "
+                "this coordinate and the browser will display the coordinate."
+            },
+            {
+                "setVerticalFov",
+                &skybrowser::luascriptfunctions::setVerticalFov,
+                "string, float",
+                "Takes an identifier to a sky browser or a sky target and a vertical "
+                "field of view. Changes the field of view as specified by the input."
+            },
+            {
+                "setBorderColor",
+                &skybrowser::luascriptfunctions::setBorderColor,
+                "string, int, int, int",
+                "Takes an identifier to a sky browser or a sky target and a rgb color "
+                "in the ranges [0, 255]."
+            },
+            {
+                "setScreenSpaceSize",
+                &skybrowser::luascriptfunctions::setScreenSpaceSize,
+                "string, float, float",
+                "Sets the screen space size of the sky browser to the numbers specified "
+                "by the input [x, y]."
+            },
+            {
+                "startSetup",
+                &skybrowser::luascriptfunctions::startSetup,
+                "",
+                "Starts the setup process of the sky browers. This function calls "
+                "the lua function 'sendOutIdsToBrowsers' in all nodes in the cluster."
+            },
+            {
+                "translateScreenSpaceRenderable",
+                &skybrowser::luascriptfunctions::translateScreenSpaceRenderable,
+                "string, float, float, float, float",
```

Comment:
This could probably be combined into just 3 inputs: a string, vec2 and vec2

example: `(name, { xstart, ystart }, { xmove, ymove })`